runtime-sdk/src/modules/consensus/mod.rs

```diff
@@ -237,6 +271,7 @@ impl API for Module {
         hook: MessageEventHookInvocation,
     ) -> Result<(), Error> {
         Self::ensure_consensus_denomination(ctx, amount.denomination())?;
+        let amount = Self::amount_to_consensus(ctx, amount.amount())?;
```

Comment:
So reclaim escrow actually accepts the number of shares to reclaim, I assume this shouldn't actually be scaled?

Maybe the argument should be renamed from `amount` to `shares` to make this a bit clearer. Hm it also shouldn't be of the denomination type probably?

Note: this method isn't used by anything yet, just exposes a wapper to emit the reclaim escrow message.

Reply:
Oh, yeah that's right, the number of shares should probably just be a plain `u128`? Let me fix that.

Reply:
Updated.

Reply:
Looks good now, thanks :+1: